resources/run_adl.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
+
+NAME=`basename "$0"`
+if [[ $SKIP_RUN_ADL -eq 1 ]]; then
+  echo "Skipping $NAME"
+  exit 0
+fi
+
+ARG=`echo $1 | tr '[:upper:]' '[:lower:]'`
```

Comment:
Should we document how this script is intended to be run? Like what this and possibly other arguments are supposed to represent?

Reply:
Added a description to the beginning of the script and in the README